src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -52,71 +51,42 @@ function ReportActionItemEmojiReactions(props) {
-        return (oldestUserReactionTimestamp || emojiReaction.createdAt) + emojiName;
-    });
-
-    const formattedReactions = _.map(sortedReactions, (reaction) => {
-        const reactionEmojiName = reaction.emojiName;
-        const usersWithReactions = _.pick(reaction.users, _.identity);
-        let reactionCount = 0;
-
-        // Loop through the users who have reacted and see how many skintones they reacted with so that we get the total count
-        _.forEach(usersWithReactions, (user) => {
-            reactionCount += _.size(user.skinTones);
-        });
-        if (!reactionCount) {
-            return null;
-        }
-        totalReactionCount += reactionCount;
-        const emojiAsset = EmojiUtils.findEmojiByName(reactionEmojiName);
-        const emojiCodes = EmojiUtils.getUniqueEmojiCodes(emojiAsset, reaction.users);
-        const hasUserReacted = Report.hasAccountIDEmojiReacted(props.currentUserPersonalDetails.accountID, reaction.users);
-        const reactionUsers = _.keys(usersWithReactions);
-        const reactionUserAccountIDs = _.map(reactionUsers, Number);
-
-        const onPress = () => {
-            props.toggleReaction(emojiAsset);
-        };
-
-        const onReactionListOpen = (event) => {
-            reactionListRef.current.showReactionList(event, popoverReactionListAnchors.current[reactionEmojiName], reactionEmojiName, reportActionID);
-        };
-
-        return {
-            reactionEmojiName,
-            emojiCodes,
-            reactionUserAccountIDs,
-            onPress,
-            reactionUsers,
-            reactionCount,
-            hasUserReacted,
-            onReactionListOpen,
-            pendingAction: reaction.pendingAction,
-        };
-    });
+    const formattedReactions = _.chain(props.emojiReactions)
+        .map((emojiReaction, emojiName) => {
+            const {emoji, emojiCodes, reactionCount, hasUserReacted, userAccountIDs, oldestTimestamp} = EmojiUtils.getEmojiReactionDetails(
+                emojiName,
+                emojiReaction,
+                props.currentUserPersonalDetails.accountID,
+            );
+
+            if (reactionCount === 0) {
+                return null;
+            }
+            totalReactionCount += reactionCount;
+
+            const onPress = () => {
+                props.toggleReaction(emoji);
+            };
+
+            const onReactionListOpen = (event) => {
+                reactionListRef.current.showReactionList(event, popoverReactionListAnchors.current[emojiName], emojiName, reportActionID);
+            };
+
+            return {
+                reactionEmojiName: emojiName,
+                emojiCodes,
+                userAccountIDs,
+                reactionCount,
+                hasUserReacted,
+                onPress,
+                onReactionListOpen,
+                pendingAction: emojiReaction.pendingAction,
+                oldestTimestamp: lodashGet(oldestTimestamp, 'oldestTimestamp'),
```

Comment:
This value is returning undefined.

Reply:
You're right, it was missed during refactoring 🤦 
Fixed now, thanks.